roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -171,16 +171,19 @@ def required_images(self):
                 required.add(self._registry_console_image(image_tag, image_info))
 
         # images for containerized components
-        if self.get_var("openshift_is_containerized"):
-            components = set()
+        if self.get_var("openshift_is_containerized", convert=bool):
+            def_img = lambda name: "{}/{}:{}".format(image_info["namespace"], name, image_tag)  # noqa
```

Comment:
Let's not use a lamba, let's just define a function like normal.

Reply:
That turns it into 5 lines (2 spaces, def, doc, and content)  instead of 1, with no real improvement. Like https://stackoverflow.com/a/37489941/262768 I prefer to break PEP8 here, but I can make it compliant if you insist.

Reply:
While I sympathize with total LOC, this lambda is just hard to grok and I definitely prefer readability over saving a couple \n's

Reply:
I don't think it benefits readability. But no matter, I saw that I could actually DRY this up a little better anyway so it ends well :)